website/docs/content/writing-style/index.md

```diff
@@ -0,0 +1,313 @@
+
+!!! Do
+
+![Two buttons, one labeled "Add user" with a leading user icon and the other labeled "Remove user" with a leading trash icon.](/assets/content/writing-style/writing-style-consistent-termonology-CTA-add-remove-do.png)
+
+!!!
+
+##### Edit and View
+
+Use when editing or viewing an entity, e.g. "Edit project" or "View project," where the project will be modified for accuracy or previewed for quality assurance.
+
+!!! Do
+
+![A menu with the actions "Edit project" paired with a pencil icon and "View project" paired with an eye icon.](/assets/content/writing-style/writing-style-consistent-termonology-menu-actions-do.png)
+
+!!!
+
+!!! Dont
+
+This is missing the object (noun) the user is doing via the action (verb). What is the user going to "View"?
+
+![A menu with the actions "Edit name" and "View" and no icons.](/assets/content/writing-style/writing-style-consistent-termonology-menu-actions-dont.png)
+
+!!!
+
+When editing an object, use "Save" for the action label. When applying another action, use an imperative, present-tense verb.
+
+!!! Do
+
+![A modal that has a form to edit an organization. The title is "Edit organization name" and the save button is "Save organization".](/assets/content/writing-style/writing-style-consistent-termonology-CTA-save-do.png)
+
+![A form to invite users by inputting their emails](/assets/content/writing-style/writing-style-consistent-termonology-CTA-invite-do.png)
+
+!!!
+
+!!! Dont
+
+![A form to edit a service principal. The title is "Edit service principal" and the save button is "Apply changes".](/assets/content/writing-style/writing-style-consistent-termonology-CTA-apply-dont.png)
+
+!!!
+
+## Punctuation
+
+### Serial/Oxford comma
+
+Use the [serial (“Oxford”)](https://www.grammarly.com/blog/punctuation-capitalization/what-is-the-oxford-comma/) comma in lists.
+
+!!! Do
+
+Choose your cluster tier, size, and network.
+
+!!!
+
+!!! Dont
+
+Choose your cluster tier, size and network.
+
+!!!
+
+### Exclamation points
+
+Avoid using exclamation points. Only use it if you’re connecting with understood, strong reader emotion. 
+
+!!! Do
+
+Achieving this centralization is a huge improvement in security posture, but it’s not the end of the journey. This is because applications don't keep secrets! It turns out, most applications do a worse job keeping secrets than our close friends.
+
+!!!
+
+!!! Dont
+
+We are excited to announce Vault 1.3!
+
+!!!
+
+### Percentages
+
+Use the % symbol instead of spelling out "percent"; e.g. , `50%`.
+
+![A card showing usage data, along side its status and name](/assets/content/writing-style/writing-style-punctuation-percentage.png)
+
+### Dashes
+
+Use an en dash (–) to indicate a range or span of numbers.
+
+![Text saying "Event dates Sep 20 – Sep 24, 2020"](/assets/content/writing-style/writing-style-punctuation-en-dashes.png)
```

Comment:
There should not be regular spaces before and after the en dash
```suggestion
![Text saying "Event dates Sep 20–Sep 24, 2020"](/assets/content/writing-style/writing-style-punctuation-en-dashes.png)
```

Reply:
Since its just normal text, you can just have the alt text be what it says.

```suggestion
![Event dates Sep 20–Sep 24, 2020](/assets/content/writing-style/writing-style-punctuation-en-dashes.png)
```